src/MoBi.UI/Views/EditParameterListView.cs

```diff
@@ -74,6 +76,10 @@ public EditParameterListView(PathElementsBinder<ParameterDTO> pathBinder,
          _gridControl.ToolTipController = toolTipController;
       }
 
+      private RepositoryItem buildingBlockRepository(ParameterDTO parameterDTO) =>
+         new UxRepositoryItemImageComboBox(_gridView, _imageListRetriever).
+            AddItem(parameterDTO.SourceReference?.BuildingBlock?.Name ?? string.Empty, parameterDTO.SourceReference?.BuildingBlock?.Icon ?? null);
```

Comment:
Could we do this null check once only and assign it to a variable. I think it would be easier to read.
`parameterDTO.SourceReference?.BuildingBlock?`

```
var buildingBlock = parameterDTO.SourceReference?.BuildingBlock;
if(buildingBlock == null)
  ...
else
  ...
```